conf/bk_server.conf

```diff
@@ -260,6 +260,9 @@ httpServerClass=org.apache.bookkeeper.http.vertx.VertxHttpServer
 # Tls certificate files refresh duration in seconds.
 # tlsCertFilesRefreshDurationSeconds=0
 
+# Enable/Disable hostname verification for tls connection.
+#tlsHostnameVerificationEnabled=false
```

Comment:
Is this a client parameter or a server parameter? Or both?

Reply:
yes, it's only client config so, it shouldn't be here. Let me remove it.